src/components/DpSearchField/DpSearchField.stories.mdx

```diff
@@ -0,0 +1,28 @@
+import { Meta } from '@storybook/addon-docs'
+import DpSearchField from './DpSearchField'
+
+<Meta
+    title="Components/SearchField"
+    component={DpSearchField}
+/>
+
+# SearchField
+
+The DpSearchField component to add input field with attribute `type="search"` and a button to submit search processing.
+
+The DpSearchField component has two events `search & reset`.
+
+## Props:
+
+### inputWidth:
+
+Value has to be a css class like 'u-1-of-2'.
+
+## Default usage
+
+```html
+<dp-search-field
+    input-width="u-1-of-2"
```

Comment:
Please let's deprecate `inputWidth`. We should instead change DpSearchInput in a way that it works in all display contexts, without the need to set an inner class like that.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/492/commits/969fb0c587ad25677a50fff83f41fc0f452b582e

Reply:
could you also add the deprecation in DpSearchField? Like src/components/DpInput/DpInput.vue:220

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/492/commits/dfb401bb777a0863e0013e3bb2711c4d59b10ea1